cms/bundles/viewsets/bundle.py

```diff
@@ -234,29 +224,44 @@ def get_scheduled_publication_display_value(self) -> str:
     def get_release_calendar_page_display_value(self) -> str:
         """Returns the release calendar page link if it exists."""
         if self.object.release_calendar_page:
+            if self.object.status == BundleStatus.PUBLISHED:
+                url = self.object.release_calendar_page.get_url(request=self.request)
+            else:
+                url = reverse("bundles:preview_release_calendar", args=[self.object.pk])
+
             return format_html(
                 '<a href="{}" target="_blank" rel="noopener">{}</a>',
-                reverse("bundles:preview_release_calendar", args=[self.object.pk]),
+                url,
                 self.object.release_calendar_page.get_admin_display_title(),
             )
         return "N/A"
 
     def get_pages_for_manager(self) -> "SafeString":
```

Comment:
It would be nice to see a docstring here with some context to decrease the cognitive load.